### ISSUE TITLE
XR Dinosaurs example in AR mode doesn't work correctly

### ISSUE THREAD

**Issue by takahirox** [MozillaReality/webxr-ios-js#98]
https://www.xrdinosaurs.com/

![image](https://user-images.githubusercontent.com/7637832/81097029-23922e80-8ebc-11ea-8354-dee4f8fd5413.png)

XR Dinosaurs example made by Toji in AR mode should render a dinosaur on Camera. But our viewer doesn't show camera. I guess it doesn't clear a scene background?

**Comment by takahirox** [comment]
In the example, document body background is defined in an [external CSS file](https://github.com/toji/xr-dinosaurs/blob/master/css/dinosaurs.css
) as

```
body {
  ...
  background-image: url('../media/textures/splash.jpg');
  ...
}
```

We override dom elements style [here](https://github.com/MozillaReality/webxr-ios-js/blob/develop/src/arkit/ARKitDevice.js#L384-L395) to hide them but we didn't take care of body.background-image yet.

```javascript
document.body.style.backgroundColor = "transparent";

var children = document.body.children;
for (var i = 0; i < children.length; i++) {
  var child = children[i];
  if (child != this._wrapperDiv && child != canvas) {
    var display = child.style.display;
    child._displayChanged = true;
    child._displayWas = display
    child.style.display = "none"
  }
}
```

I'm trying to resolve this issue but one problem here. `document.body.style.backgroundImage = ""` has no effect. It seems CSS rules defined in an external CSS file are stored in `document.styleSheets[]`.  So in the example I needed to find a rule from the array and override. Does anyone know if there is any easier way to override?

**Comment by takahirox** [comment]
I found `document.body.style.backgroundImage = "none"` seems to work.

**Comment by takahirox** [comment]
Fixed in #99
